app/components/show-header.js

```diff
@@ -0,0 +1,45 @@
+import Component from '@ember/component';
```

Comment:
would you mind using two spaces for indentations? it's purely stylistic, but it's the house style here